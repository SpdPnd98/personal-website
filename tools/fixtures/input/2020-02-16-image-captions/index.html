<p>Hello this is some text, followed by</p>

<p><img src="20200216_1.png" style="max-width: 400px; width: 100%; margin: 0 auto; display: block;" alt="Interesting Image" /></p>
<p class="text-center text-gray lh-condensed-ultra f6">Caption is a thing</p>

<p>Then I talk a little more about other things, which will follow by</p>

<p><img src="20200216_2.png" style="max-width: 400px; width: 100%; margin: 0 auto; display: block;" alt="Another Image" /></p>
<p class="text-center text-gray lh-condensed-ultra f6">youtube is clickbait</p>
