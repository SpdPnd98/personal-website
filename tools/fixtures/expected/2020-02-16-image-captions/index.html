<p>Hello this is some text, followed by</p>

<p><img src="test1.png" style="max-width: 400px; width: 100%; margin: 0 auto; display: block;" alt="a test" /></p>
<p class="text-center text-gray lh-condensed-ultra f6">yup, captions</p>

<p>Then I talk a little more about other things, which will follow by</p>

<p><img src="test2.png" style="max-width: 400px; width: 100%; margin: 0 auto; display: block;" alt="a test 2" /></p>
<p class="text-center text-gray lh-condensed-ultra f6">yay captions</p>
